packages/expo-video/ios/VideoView.swift

```diff
@@ -71,6 +83,13 @@ public final class VideoView: ExpoView, AVPlayerViewControllerDelegate {
 
     if playerViewController.responds(to: selectorToForceFullScreenMode) {
       playerViewController.perform(selectorToForceFullScreenMode, with: true, with: nil)
+    } else {
+      #if os(tvOS)
+      // For TV, save the currently playing state, and present the view controller normally
+      wasPlaying = player?.pointer.timeControlStatus == .playing
```

Comment:
```suggestion
      wasPlaying = player?.isPlaying == true
```

Reply:
@behenate this is the same code that is used in the iOS `willEndFullScreenPresentationWithAnimationCoordinator` method, see line 164

Reply:
I made this change, plus a few others to improve full screen transitions on TV.